docs/configuration/avro.asciidoc

```diff
@@ -0,0 +1,65 @@
+= Avro Serialization
+:awestruct-layout: doc
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+Debezium by default serializes data events into a free-form JSON format. It is possible to use https://avro.apache.org/[Avro] serialization that will use binary encoding for the data and associate a schema identifier with the message to allow safe schema evolution over a longer time period.
+
+Debezium plug-ins will typically use Avro serializations in two different scenarios
+
+ * Use pre-configured http://debezium.io/docs/docker/[Debezium Docker images]
+ * Deploy Debezium plug-ins into https://www.confluent.io/product/confluent-open-source/[Confluent Open Source] Kafka distribution
+
+== Technical information
+A system that wants to use Avro serialization needs to complete two steps
+
+* Deploy a https://github.com/confluentinc/schema-registry[Schema Registry] instance
+* Use these properties to configure Apache Connect instance
+[source]
+----
+key.converter=io.confluent.connect.avro.AvroConverter
+key.converter.schema.registry.url=http://localhost:8081
+value.converter=io.confluent.connect.avro.AvroConverter
+value.converter.schema.registry.url=http://localhost:8081
+----
+Note: In addition to setting key/value converters it is necessary to set internal key/value converters due to a https://issues.apache.org/jira/browse/KAFKA-3988[known issue].
```

Comment:
Actually, we recommend to _always_ use JSON converters for the internal keys and values. The volume of messages is small, and the readability helps if you ever need to see/edit the stored configurations and offsets.